### PR TITLE
Add support for sslmode on external databases.   Fixes #11

### LINKED ISSUE #11: Add support for Specifying SSL Mode for Postgres Connection
Add support to configure the sslmode boolean value for the Postgres DB connection string.

### PR DESCRIPTION
**Pull request by mcowger**
* Added support to declare the sslmode for an external database.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,6 +119,7 @@ The following tables lists the configurable parameters of the Harbor chart and t
 | `database.external.username` | The username of external database | `user` |
 | `database.external.password` | The password of external database | `password` |
 | `database.external.coreDatabase` | The database used by core service | `registry` |
+| `database.external.sslmode` | Connection method of external database (require|prefer|disable) | `disable`|
```
Comment (ywk253100) [inline README.md]: The default value doesn't match with the one in `values.yaml`.

**Comment by ywk253100** [comment]
@mcowger Thank you for your contribution!

There are four components using the database in Harbor: Clair, Notary server, Notary signer and Harbor core(configured in adminserver).
The first three have been covered by your PR. But the last one doesn't. So we need to set it in the configmap of adminserver https://github.com/goharbor/harbor-helm/blob/master/templates/adminserver/adminserver-cm.yaml#L9.

And currently the `sslmode` is hardcoded as `false` in the Harbor core source code, I have opened an issue to track this https://github.com/goharbor/harbor/issues/5843.

**Comment by mcowger** [comment]
Thanks for the catch!  I've updated the PR to address that.

File: README.md (hunk 1 above)
Comment (mcowger) [reply README.md]: Thanks.  Corrected.

**Pull request merged**